### ISSUE TITLE
Improve interoperability with Solana Ledger app

### ISSUE THREAD

**Issue by t-nelson** [project-serum/spl-token-wallet#152]
[Sollet.io currently uses](https://github.com/project-serum/spl-token-wallet/blob/e1588a6dfd6dcb6f9e2589b572ed95241e69a6cd/src/utils/tokens/instructions.js#L32-L36) the SPL Token `Transfer` instruction to send SPL Tokens between user accounts.  Unfortunately, that instruction lacks introspectability required to be effectively verified in situations where the cluster cannot be queried, such as offline and/or hardware wallet signing.  To solve this issue a new instruction, [`TransferChecked`](https://github.com/solana-labs/solana-program-library/blob/5df346163f4e63132af850c300a085418a2cb44d/token/program/src/instruction.rs#L245-L273), was introduced, which carries the address and decimals of the token's mint.  This allows for ticker symbol lookup and proper amount formatting without querying the mint from the cluster,  These values are also asserted during execution to prevent spoofing. Due to `Transfer`'s lack of verifiability it isn't, and won't be, supported on Solana's Ledger app. So continuing to use this instruction will prevent Sollet's Ledger compatibility.  Please switch to `TransferChecked`!

TLDR; The SPL Token `Transfer` instruction sucks.  Please switch to the `TransferChecked` instruction

**Comment by armaniferrante** [comment]
Addressed by https://github.com/project-serum/spl-token-wallet/pull/191.

**Comment by t-nelson** [comment]
Nice!  Thanks @armaniferrante!

These transfers should have proper, reviewable confirmation on Ledger devices once v1.0.5 of the Solana app hits Ledger Live.  The release is tagged and rattling around in LedgerHQ's release pipeline somewhere.  I expect it to pop out within a few days :crossed_fingers:

**Comment by t-nelson** [comment]
>  I expect it to pop out within a few days :crossed_fingers:

New Ledger app finally dropped.  The blind signing setting is no longer required for SPL Token transfers on Sollet, with or without ATA create instructions

**Comment by armaniferrante** [comment]
Sweet thanks for the update!
